neo4j/src/main/java/eu/drus/jpa/unit/neo4j/ext/HibernateOgmConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package eu.drus.jpa.unit.neo4j.ext;
+
+import java.io.IOException;
+import java.util.Map;
+
+import eu.drus.jpa.unit.api.JpaUnitException;
+import eu.drus.jpa.unit.spi.PersistenceUnitDescriptor;
+
+public class HibernateOgmConfiguration extends AbstractConfiguration {
+
+    protected static final String HIBERNATE_OGM_DATASTORE_PASSWORD = "hibernate.ogm.datastore.password";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)

Reply:
false positive. This is just a key word